src/components/Todo.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import User from './User';
+
+function Todo({ todo: { user, title, completed } }) {
+  return (
+    <>
+      <div className={completed
+        ? 'todo todo__completed'
+        : 'todo'}
+      >
+        <p><User user={user} /></p>
```

Comment:
```suggestion
        <p>
          <User {...user} />
        </p>
```
Павило один тег - одна линия никто не отменял

Reply:
И старайся не писать три одинаковых слова подряд, как <User user={user} />. Используй деструктуризацию или давай пропсам другое имя